src/jcsclient/compute.py

```diff
@@ -147,7 +147,7 @@ def start_instances(self, args):
         """
         return instance.start_instances(self.url, self.verb,
                                         self.headers, self.version, args)
- 
```

Comment:
the setting in my VIM editor shows trailing whitespaces in red. I have only removed them. 
The setting takes into regard a bare minimum indentation and white spaces check.